<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceSelectView</class>
 <widget class="QDialog" name="DeviceSelectView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>84</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Device selection</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Devices :</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="devicesSelComboBox"/>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="selCamCancelBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="selCamOkBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>selCamOkBtn</sender>
   <signal>clicked()</signal>
   <receiver>DeviceSelectView</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selCamCancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>DeviceSelectView</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
